nservicebus/rabbitmq/configuration-api.md

```diff
@@ -107,4 +107,25 @@ If the routing topologies mentioned above isn't flexible enough then take full c
  1. Define the topology by creating a class implementing `IRoutingTopology`
  1. Register it with the transport calling `.UseRoutingTopology` as shown below
 
-snippet:rabbitmq-config-useroutingtopology
\ No newline at end of file
+snippet:rabbitmq-config-useroutingtopology
+
+## Transactions and delivery guarantees
+
+### Version 6 and above
+
+The RabbitMQ transport supports the following [Transport Transaction Modes](/nservicebus/messaging/transactions.md):
+
+* Receive Only
+* No transactions
+
+### Receive Only
+
+When running in Receive Only mode, the RabbitMQ transport consumes messages from the broker in manual acknowledgment mode. After a message is successfully processed, it is acknowledged via the AMQP [basic.ack](http://www.rabbitmq.com/amqp-0-9-1-quickref.html#basic.ack) method, which lets the broker know that the message can be removed from the queue. If a message is not successfully processed and needs to be retried, it is requeued via the AMQP [basic.reject](http://www.rabbitmq.com/amqp-0-9-1-quickref.html#basic.reject) method.
+
+If the connection to the broker is lost for any reason before a message can be acknowledged, the message will automatically be requeued by the broker, letting it be retried by the endpoint.
```

Comment:
Can we highlight the danger of this better? For example something like:

If the connection to the broker is lost for any reason before a message can be acknowledged even if the message was successfully processed, the message will still be automatically be requeued by the broker. This will result in the endpoint processing the same message multiple times.